fire_atlas_editor/lib/store/actions/atlas_actions.dart

```diff
@@ -100,15 +110,16 @@ class RemoveSelectedSelectionAction extends MicroStoreAction<FireAtlasState> {
 class SaveAction extends MicroStoreAction<FireAtlasState> {
   @override
   FireAtlasState perform(FireAtlasState state) {
-    FireAtlasStorage.saveProject(state.currentAtlas);
-
-    state.hasChanges = false;
+    if (state.currentAtlas != null) {
+      FireAtlasStorage.saveProject(state.currentAtlas!);
```

Comment:
if you save to a variable you can avoid the !

Reply:
I use the ! only on this line on this method, I feel it is not worth to add a new line just to avoid the !